src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,36 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { User } from '../../types/User';
+import { Todo } from '../../types/Todo';
+import cn from 'classnames';
+import { getUser } from '../../api';
+
+interface Props {
+  todo: Todo;
+  reset: () => void;
+}
+
+export const TodoModal: React.FC<Props> = ({ todo, reset }) => {
+  const [isLoading, setIsLoading] = useState(true);
+  const [user, setUser] = useState<User | null>(null);
+
+  useEffect(() => {
+    const fetchUser = async () => {
+      setIsLoading(true);
+      const person = await getUser(todo.userId);
+
+      setUser(person);
+      setIsLoading(false);
+    };
+
+    fetchUser();
+  }, []);
```

Comment:
}, [todo.userId]);

Reply:
![image](https://github.com/mate-academy/react_dynamic-list-of-todos/assets/43861663/83fb6969-797d-4a78-8a37-946f3c5559bb)